prompting/tasks/task_registry.py

```diff
@@ -24,15 +26,24 @@ class TaskConfig(BaseModel):
 
 class TaskRegistry(BaseModel):
     task_configs: ClassVar[list[TaskConfig]] = [
-        TaskConfig(task=QuestionAnsweringTask, probability=0.55, datasets=[WikiDataset], reward_model=QARewardConfig),
+        TaskConfig(task=QuestionAnsweringTask, probability=0.2, datasets=[WikiDataset], reward_model=QARewardConfig),
```

Comment:
We should gradually change probabilities, to make network more stable during the releases

Reply:
Yeah, that's fair. What percentages would you suggest?